context.go

```diff
@@ -18,27 +18,24 @@ import (
 
 // Ctx defines the supporting context of the tool.
 type Ctx struct {
-	GOPATH  string   // Selected Go path
-	GOPATHS []string // Other Go paths
+	GOPATH     string   // Selected Go path
+	GOPATHS    []string // Other Go paths
+	WorkingDir string
 }
 
 // NewContext creates a struct with the project's GOPATH. It assumes
 // that of your "GOPATH"'s we want the one we are currently in.
-func NewContext() (*Ctx, error) {
-	// this way we get the default GOPATH that was added in 1.8
-	buildContext := build.Default
-	wd, err := os.Getwd()
+func NewContext(wd string, env []string) (*Ctx, error) {
```

Comment:
Then my unfamiliarity with the code base worked to my advantage, because I started by targeting general parity with the `exec.Cmd` API in use since I didn't know the minimum input requirements  :laughing: